jenkins2/Dockerfile

```diff
@@ -0,0 +1,115 @@
+FROM registry.service.opg.digital/opguk/jre-8
+
+#Install tools
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add - && \
+    add-apt-repository \
+      "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+      $(lsb_release -cs) \
+      stable" && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends \
+      apparmor \
+      apt-transport-https \
+      ca-certificates \
+      curl \
+      docker-ce \
+      fontconfig \
+      gettext \
+      libcurl4-gnutls-dev \
+      libexpat1-dev \
+      libffi-dev \
+      libgmp3-dev \
```

Comment:
Not sure if all the packages are needed, as I've remove meta-compose and docker-machine from the container.